PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -252,6 +252,24 @@ $ProdToFullName = @{
     OneDrive = "OneDrive for Business";
 }
 
+function Get-FileEncoding{
+    <#
+    .Description
+    This function returns encoding type for setting content.
+    .Functionality
+    Internal
+    #>
+    $PSVersion = $PSVersionTable.PSVersion
+
+    $Encoding = 'utf8'
+
+    if ($PSVersion -ge '7.2.0'){
```

Comment:
It appears that powershell 6.0, 7.0 and 7.1 also default to (and supported) utf8NoBom.  Even though they are out of support, I suggest changing this check  emit utf8NoBom for those versions as well.

Reply:
I updated to set encoding to utf8NoBom if PowerShell version is 6.0 or greater.